File: backend/src/main/java/core/dto/ProductDTO.java
package core.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ProductDTO {
    private Long id;
    private String name;
    private int kcal;
    private String unit;
    private String createdBy;
    private LocalDateTime createdAt;
}

File: android/app/src/main/java/tk/ssau/fitnesstko/model/dto/nutrition/ProductDTO.kt
package tk.ssau.fitnesstko.model.dto.nutrition

import java.math.BigDecimal

data class ProductDTO(
    val id: Long?,
    val name: String,
    val kcal: Int,
    val unit: String,
    val grams: BigDecimal,
    val portion: BigDecimal,
    val createdAt: String?,
    val percent: Int?,
    val fats: BigDecimal,
    val carbs: BigDecimal,
    val proteins: BigDecimal
)
